dev-db/redis/redis-3.0.1.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit autotools eutils flag-o-matic systemd toolchain-funcs user
+
+DESCRIPTION="A persistent caching system, key-value and data structures database"
+HOMEPAGE="http://redis.io/"
+SRC_URI="http://download.redis.io/releases/${P}.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64 ~hppa ~ppc64 ~x86 ~amd64-linux ~x86-linux ~x86-macos ~x86-solaris"
+IUSE="+jemalloc tcmalloc test"
+SLOT="0"
+
+RDEPEND=">=dev-lang/lua-5.1:*
+	tcmalloc? ( dev-util/google-perftools )
+	jemalloc? ( >=dev-libs/jemalloc-3.2 )"
+DEPEND="virtual/pkgconfig
+	>=sys-devel/autoconf-2.63
+	test? ( dev-lang/tcl:0= )
+	${RDEPEND}"
+REQUIRED_USE="?? ( tcmalloc jemalloc )"
+
+S="${WORKDIR}/${PN}-${PV/_/-}"
+
+pkg_setup() {
+	enewgroup redis 75
+	enewuser redis 75 -1 /var/lib/redis redis
+}
+
+src_prepare() {
+	epatch "${FILESDIR}"/${PN}-2.8.3-shared.patch
+	epatch "${FILESDIR}"/${PN}-2.8.17-config.patch
+	epatch "${FILESDIR}"/${PN}-3.0.0-sharedlua.patch
+
+	# Copy lua modules into build dir
+	cp "${S}"/deps/lua/src/{fpconv,lua_bit,lua_cjson,lua_cmsgpack,lua_struct,strbuf}.c "${S}"/src || die
+	cp "${S}"/deps/lua/src/{fpconv,strbuf}.h "${S}"/src || die
+	# Append cflag for lua_cjson
+	# https://github.com/antirez/redis/commit/4fdcd213#diff-3ba529ae517f6b57803af0502f52a40bL61
+	append-cflags "-DENABLE_CJSON_GLOBAL"
+
+	# now we will rewrite present Makefiles
+	local makefiles=""
+	for MKF in $(find -name 'Makefile' | cut -b 3-); do
```

Comment:
Nope, not going to like hackery like that. What's the deal with cutting here anyway? Use proper `while read; ...; done < <(...)`. Look at python-utils-r1 how use `find` like that.